python/test/unit/hopper/test_persistent_warp_specialized_fused-attention.py

```diff
@@ -32,6 +32,14 @@
 import triton.language as tl
 
 
+def is_hip():
+    return triton.runtime.driver.active.get_current_target()[0] == "hip"
+
+
+def get_target():
```

Comment:
Just unify these two funtions into one for now? It's simpiler and concise that way. We can use two if there is need arise in the future. For now not necessary.

```
target = triton.runtime.driver.active.get_current_target()
return target[0] == "hip" && target[1] == "gfx90a"
```